### PR TITLE
CI test suite

### PR DESCRIPTION
**Pull request by why-not-try-calmer**
The official test suite is not CI-ready; it uses a web client requiring manual input:

Capture d’écran du 2023-04-03 10-58-50

The wrapper script proposed here should make our lives a bit easier.

### CODE REVIEW

**Comment by why-not-try-calmer** [comment]
This actually needs more debugging. WIP.

File: docker-compose.tests.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+services:
+  conformance_suite:
+    image: ogccite/ets-ogcapi-features10
+    depends_on: [django]
+
+  test_client:
+    build: ${PWD}/conformance-tests
```
Comment (olivierdalang) [inline docker-compose.tests.yml:7]: paths are relative in docker compose already (you can just use ./conformance-tests)

File: docker-compose.tests.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    volumes:
+      - ${PWD}/conformance-tests:/conformance-tests
```
Comment (olivierdalang) [inline docker-compose.tests.yml:9]: same

File: conformance-tests/ogcf.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import unittest
+from urllib.parse import quote_plus, urlencode
+
+import requests
+
+
+class ConformanceTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        app_service = "django"
```
Comment (olivierdalang) [inline conformance-tests/ogcf.py:10]: django:8000

**Review by olivierdalang — CHANGES_REQUESTED** [review]

**Comment by olivierdalang** [comment]
Changed the approach to use directly the CLI. Unfortunately their provided Docker image doesn't contain the CLI executable. Fortunately someone did the work already: https://github.com/opengeospatial/ets-ogcapi-features10/issues/197

I also added the test at the end of the already existing quickstart test, as anyway the test always succeed for now because it exits with 0 even if there are issues. We can improve that once the test result start to be meaningful (e.g. once we're more or less conform).

The CI uploads the result to the artifacts (current run: https://github.com/opengisch/signalo-oapif/suites/12019989528/artifacts/631510246)

**Review by olivierdalang — APPROVED** [review]

**Review by olivierdalang — APPROVED** [review]

**Pull request merged**